modules/osdk-multi-arch-node-affinity.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-multi-arch-support.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-multi-arch-node-affinity_{context}"]
+= Safe scheduling an Operator's node affinity for multiple architecture compute nodes
+
+Node affinity is a set of rules used by the scheduler to define a pod's placement. Set node affinity requirements to ensure that your Operator's workloads only schedule to compute nodes with architectures that are compatible with the images in the pod. 
+
+[WARNING]
+====
+If you do not set node affinity requirements and a container is scheduled to a compute node with an incompatible architecture, the pod fails with an `exec format` error. When the pod restarts and fails again with the same error, an `ImagePullBackoff` event occurs.
```

Comment:
The team explained the subtlety in the error messages today to me.
The `exec format` error:
    occurs when image is a single manifest image which is run on a node with incompatible architecture
    
The `image pull back-off` error:
    occurs when the image is manifestlisted but does not contain the image manifest matching the node architecture

Reply:
The possible events for workloads are:

- `CrashLoopBackOff`: the Entrypoint will fail to run by showing `exec format error` in the logs (the image is a standard manifest);
- `ImagePullBackOff`: the image is a manifest list, but there is no manifest for the node's architecture on which the Pod is placed or the wrong node affinity terms are set.

Reply:
@jaypoulz and @aleskandro 
Thanks for the feedback. I addressed this in https://github.com/openshift/openshift-docs/pull/64355/commits/11fca674b5bcb691552fd3f3717485f684e9ceb9. PTAL.

Reply:
Thanks for the feedback @jaypoulz and @aleskandro 

I addressed your feedback in https://github.com/openshift/openshift-docs/pull/64355/commits/ebbc2832c259310e7feb82c9c59e9c30b1e20ead

PTAL at the update for accuracy.